### ISSUE TITLE
Memory usage is not reading the value from the kernel, therefore it's not correct

### ISSUE THREAD

**Issue by fulalas** [oguzhaninan/Stacer#525]
![Screenshot_2023-04-25_20-54-14](https://user-images.githubusercontent.com/27843666/234401268-2fdc6281-d429-4720-9632-f061a7c65364.png)

**Comment by priyankarpal** [comment]
yes. I also faced this issue. even after cleaning up caches the software automatically got deleted idk why

**Comment by QuentiumYT** [comment]
Hello, I am maintaining a fork of Stacer actively https://github.com/QuentiumYT/Stacer/ as a continuation of this original project (fixing bugs & adding small features, its goal is to continue the legacy of the application without changing it completely). It has been accepted into newer Debian, Ubuntu, Arch (and Fedora incoming) distros. (See https://repology.org/project/stacer/versions)

Regarding this issue, you are correct, I haven't stumbled upon that yet, will release a newer version of the app soon!

Linked to #535
